spec/engine/miq_ae_method_service/miq_ae_service_methods_spec.rb

```diff
@@ -209,7 +209,7 @@ def category_create_script
     it "create retire request" do
       allow(workspace).to receive(:disable_rbac)
       allow(Service).to receive(:find).with(service.id).and_return(service)
-      expect(service).to receive(:make_retire_request).with(user).and_return(miq_request)
+      expect(Service).to receive(:make_retire_request).with(service.id, user).and_return(miq_request)
```

Comment:
@tinaafitz can we try this with an array of classes to make sure we have covered all the classes that are retireable and also trap errors on classes that are not retireable, e.g. CloudSubnet or some other class.

Reply:
@lfu Can you add spec tests for this?